automation_oca/static/src/fields/automation_graph/automation_graph.esm.js

```diff
@@ -25,24 +24,27 @@ export class AutomationGraph extends Component {
         return {
             type: "line",
             data: {
-                labels: this.props.value.done.map(function (pt) {
+                // Labels: this.props.value.done.map(function (pt) {
+                //     return pt.x;
+                // }),
+                labels: this.props.record.data.graph_data.done.map(function (pt) {
                     return pt.x;
                 }),
                 datasets: [
                     {
                         backgroundColor: "#4CAF5080",
                         borderColor: "#4CAF50",
-                        data: this.props.value.done,
+                        data: this.props.record.data.graph_data.done,
```

Comment:
The idea of using value is reusing the widget in other fields if necessary. Why did you apply this change?